pkg/refactor/logstore/dblogstore/logstore.go

```diff
@@ -0,0 +1,279 @@
+package dblogstore
```

Comment:
To have a consistent store pattern similar to the other `store` implementations. Please move this package to `refactor/datastore`.